wandb/sdk/internal/artifacts.py

```diff
@@ -30,7 +44,13 @@ def __call__(
             pass
 
 
-def _manifest_json_from_proto(manifest: "wandb_internal_pb2.ArtifactManifest") -> Dict:
+class ArtifactCommitError(Exception):
```

Comment:
what is your sense of having this error in the same location as other errors (i.e. errors/__init__.py). Honestly, we are not consistent about it, so it is fine either way...